tests/test_functional.py

```diff
@@ -562,3 +562,14 @@ def test_obal_changelog_custom():
 def test_obal_srpm():
     assert_obal_success(['srpm', 'hello'])
     assert_mockbin_log(['tito build --srpm --offline --output {pwd}/SRPMs'])
+
+
+@obal_cli_test(repotype='upstream')
+def test_obal_mock():
+    assert_obal_success(['mock', 'hello', '--config', 'mock/el7.cfg'])
```

Comment:
`--config` is optional, right, so we could drop it from the tests?

Reply:
also, I wonder if we should attempt a test with two packages, that will ensure mock was only called once